avm/res/compute/gallery/image/main.bicep

```diff
@@ -0,0 +1,248 @@
+metadata name = 'Compute Galleries Image Definitions'
+metadata description = 'This module deploys an Azure Compute Gallery Image Definition.'
+metadata owner = 'Azure/module-maintainers'
+
+@sys.description('Required. Name of the image definition.')
+param name string
+
+@sys.description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@sys.description('Conditional. The name of the parent Azure Shared Image Gallery. Required if the template is used in a standalone deployment.')
+@minLength(1)
+param galleryName string
+
+@sys.description('Optional. OS type of the image to be created.')
+@allowed([
+  'Windows'
+  'Linux'
+])
+param osType string = 'Windows'
```

Comment:
A minor detail, but I'd suggest to switch the dafaults to a newer Linux image

Reply:
Done: https://github.com/Azure/bicep-registry-modules/pull/752/commits/5fa962caf7723f452d17a04e9e7344492e644721